kansha/card_addons/checklist/view.py

```diff
@@ -98,7 +102,7 @@ def render_Checklist(self, h, comp, model):
             else:
                 h << self.title.render(h.AsyncRenderer())
                 h << h.a(h.i(class_='icon-cross'), class_='delete').action(
-                    ajax.Update(render='deleted', action=comp.answer)
+                    ajax.Update(render='', action=comp.answer)
```

Comment:
The 'deleted' view is there to hide the component of the deleted checklist when we render aynchronously.